b2share/config.py

```diff
@@ -261,4 +261,9 @@
 
 FILES_REST_DEFAULT_MAX_FILE_SIZE = 1000 * 1024 * 1024 # 1 GB by default for RC4
 
-SITE_FUNCTION = 'testing'
+# prominently displayed on the front page, except when set to "production"
+SITE_FUNCTION = 'demo' # set to "production" on production instances
+
+# if the TRAINING_SITE_LINK parameter is not empty, a message will show up
+# on the front page redirecting the testers to this link
+TRAINING_SITE_LINK = "http://trng-b2share.eudat.eu"
```

Comment:
We need to be careful with this one. It is specific to CSC. It will be displayed on demo instances people start themselves.

It would be more appropriate to have a customizable header/footer or a "flash" message shown when the user goes to the front page.